### PR TITLE
Read long log lines from file storage correctly

### PR DESCRIPTION
**Pull request by hg**
Log lines may contain up to 1 MB of text, but with default settings `bufio.Scanner` only reads lines up to 64k in length and fails on anything larger, rendering the `Scanner` unusable.

As a result, the user won't see anything after the first very long line, including the line itself.

```yaml
when:
  event: push

steps:
  logs:
    image: ubuntu:24.04
    commands:
      - seq 1 3
      - tr -dc '[:alnum:]' </dev/urandom | head -c70k
      - echo
      - seq 4 6
```

The output should be:

```
1
2
3
zkwmLrpYmk6XcgJZiYqzNO3hsIV26HJfVQfyvSo1ccrMqP ... very long line of random text
4
5
6
```

Instead, Woodpecker cuts everything after `3`:

```
1
2
3
(end)
```

I tested this on jobs with up to 20 MBs of output (simply update `70k` with something like `20M`).

### CODE REVIEW

File: cli/exec/exec.go [hunk 1]
```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/rs/zerolog/log"
 	"github.com/urfave/cli/v3"
 
+	"go.woodpecker-ci.org/woodpecker/v2/agent"
```
Comment (6543) [inline cli/exec/exec.go:31]: if you need to import the agent then the const is in the wrong place and should be moved elsewhere ...!

I suggest [go.woodpecker-ci.org/woodpecker/v2/pipeline](https://github.com/woodpecker-ci/woodpecker/blob/main/pipeline/const.go)

**Comment by 6543** [comment]
please dont forcepush if you can! it makes reviews harder and we squashmerge anyway ...

**Review by lafriks — APPROVED** [review]

**Comment by 6543** [comment]
https://ci.woodpecker-ci.org/repos/3780/pipeline/19786/24 ... well

**Comment by 6543** [comment]
thanks for not running at the dedicated pull :/

**Comment by 6543** [comment]
If you d'like to create a pull?

**Pull request merged**